visual_tests.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
```

Comment:
I left that in for the moment, should I add it or remove it? if yeS: in a different PR? I would run this script on failure on appveyor so that the html file is included in the result_images.zip, which is IMO nice for a quick overview...